ckan/templates/user/snippets/followee_dropdown.html

```diff
@@ -20,7 +20,8 @@
     <div class="popover-header">
       <div class="input-group">
         <span class="input-group-addon"><i class="fa fa-search"></i></span>
-        <input type="text" class="form-control" name="q" placeholder="{{ _('Search list...') }}" value="{{context.q}}" autocomplete="off">
+        <label for="field-followee-search" style="display: none;">Search Followees</label>
```

Comment:
`display: none;` is not recommended as it hides the element also from screen readers [ref](https://accessibility.18f.gov/hidden-content/). Either use .sr-only class which is available in [bootstrap](https://getbootstrap.com/docs/3.4/components/#callout-inputgroup-form-labels) or add aria-label to the input field, which I would recommend as it is largely used in the current codebase.